core/io/file_access_compressed.cpp

```diff
@@ -270,8 +270,8 @@ bool FileAccessCompressed::eof_reached() const {
 
 uint8_t FileAccessCompressed::get_8() const {
 
-	ERR_FAIL_COND_V(writing, 0);
-	ERR_FAIL_COND_V(!f, 0);
+	ERR_FAIL_COND_V_MSG(writing, 0, "File has not benn opened in read mode.");
```

Comment:
`s/benn/been/` (also below)

Reply:
Not related to these changes, but I'd move this check below the `!f` one, that seems a more natural order (and matches what is done further down).